Sources/Vitamin/Components/Progressbar/VitaminProgressbar.swift

```diff
@@ -12,12 +12,16 @@ public final class VitaminProgressbar: UIView {
         variant: Variant = .circular(size: . medium, style: .empty),
         progressType: ProgressType,
         showTrack: Bool = true,
-        leftLabelText: String? = nil
+        leftLabelText: String? = nil,
+        vitaminAccessibilityLabel: String? = nil
     ) {
         super.init(frame: .zero)
         self.variant = variant
         self.showTrack = showTrack
         self.leftLabelText = leftLabelText
+        if let label = vitaminAccessibilityLabel {
+            self.vitaminAccessibilityLabel = label
+        }
```

Comment:
Based on the previous suggestion
```suggestion
        self.accessibilityLabel = accessibilityLabel
```